### PR TITLE
refactor: replace TRUE/FALSE with true/false

### PR DESCRIPTION
**Pull request by dundargoc**

### CODE REVIEW

File: src/nvim/window.c [hunk 1]
```diff
@@ -2797,7 +2804,7 @@ winframe_remove (
       }
     }
     frame_new_height(frp2, frp2->fr_height + frp_close->fr_height,
-        frp2 == frp_close->fr_next ? TRUE : FALSE, FALSE);
+                     frp2 == frp_close->fr_next ? true : false, false);
```
Comment (vigoux) [inline src/nvim/window.c:2807]: Please please pleeease...
```suggestion
                     frp2 == frp_close->fr_next, false);
```

File: src/nvim/window.c [hunk 2]
```diff
@@ -2826,7 +2833,7 @@ winframe_remove (
       }
     }
     frame_new_width(frp2, frp2->fr_width + frp_close->fr_width,
-        frp2 == frp_close->fr_next ? TRUE : FALSE, FALSE);
+                    frp2 == frp_close->fr_next ? true : false, false);
```
Comment (vigoux) [inline src/nvim/window.c:2836]:
```suggestion
                    frp2 == frp_close->fr_next, false);
```

File: src/nvim/window.c [hunk 3]
```diff
@@ -5408,16 +5405,16 @@ void win_setminwidth(void)
   }
 }
 
-/*
- * Status line of dragwin is dragged "offset" lines down (negative is up).
- */
+/// Status line of dragwin is dragged "offset" lines down (negative is up).
+///
+/// @param up  if true, drag status line up, otherwise down.
```
Comment (vigoux) [inline src/nvim/window.c:5410]: There's no `up` param ?

**Review by vigoux — CHANGES_REQUESTED** [review]

File: src/nvim/window.c (hunk 1 above)
Comment (dundargoc) [reply src/nvim/window.c]: Haha yeah, I also reacted on that but didn't want to touch it.

File: src/nvim/window.c (hunk 3 above)
Comment (dundargoc) [reply src/nvim/window.c]: Oops. Good catch!

**Review by vigoux — APPROVED** [review]

**Review by vigoux — APPROVED** [review]

File: src/nvim/window.c [hunk 4]
```diff
@@ -2982,18 +2989,14 @@ static bool frame_has_win(const frame_T *frp, const win_T *wp)
...
-    int height,
-    int topfirst,                   /* resize topmost contained frame first */
-    int wfh                        /* obey 'winfixheight' when there is a choice;
-                                   may cause the height not to be set */
-)
+/// Set a new height for a frame.  Recursively sets the height for contained
+/// frames and windows.  Caller must take care of positions.
+///
+/// @param topfirst  resize topmost contained frame first.
+/// @parma wfh       obey 'winfixheight' when there is a choice;
```
Comment (vigoux) [inline src/nvim/window.c:2996]:
```suggestion
/// @param wfh       obey 'winfixheight' when there is a choice;
```

**Review by vigoux — CHANGES_REQUESTED** [review]

**Review by vigoux — APPROVED** [review]

File: src/nvim/window.c [hunk 5]
```diff
@@ -1010,8 +1011,8 @@ int win_split_ins(int size, int flags, win_T *new_wp, int dir)
         if (frp->fr_win != oldwin && frp->fr_win != NULL
             && (frp->fr_win->w_width > new_size
                 || frp->fr_win->w_width > oldwin->w_width
-                                          - new_size - 1)) {
-          do_equal = TRUE;
+                - new_size - 1)) {
```
Comment (bfredl) [inline src/nvim/window.c:1014]: new indent is misleading, add parens around `(oldwin->w_width - new_size - 1)`

**Pull request merged**